src/nisarqa/raster_classes.py

```diff
@@ -660,9 +710,9 @@ def __post_init__(self):
         # of referencing by the upper-left corner of the pixel, but the
         # NISAR L2 product writers construct `xCoordinates` and `yCoordinates`
         # to refer to the location of the center of the pixels.
-        self.x_start = float(self.x_coordinates[0] - 0.5 * self.x_spacing)
+        self.x_start = float(self.x_coordinates[0] - 0.5 * self.x_posting)
 
-        self.x_stop = float(self.x_coordinates[-1] + 0.5 * self.x_spacing)
+        self.x_stop = float(self.x_coordinates[-1] + 0.5 * self.x_posting)
```

Comment:
Why's there a blank line between `self.x_start` and `self.x_stop`. Let's remove the blank line.